src/channel.h

```diff
@@ -36,6 +36,8 @@
 #include "protocol.h"
 #include "socket.h"
 
```

Comment:
No blank line - when I kick off the build clang-format's probably going to complain...